.github/CODEOWNERS

```diff
@@ -451,6 +451,9 @@
 # Typings.
 /typings @ankita240796
 
+# Takeout.
```

Comment:
Move this above `Typings` project. (Ordered in alphabetical manners of Project names)

Reply:
done